assets/styles/article.less

```diff
@@ -42,3 +42,24 @@ h2 a:hover, h3 a:hover, h4 a:hover {
   padding: 4px;
   vertical-align: top;
 }
+
+blockquote {
+  background: #f9f9f9;
+  border-left: 10px solid #ccc;
+  font-size: 12pt;
+  font-family: 'Inconsolata', ;
```

Comment:
It probably hasn't affected the appearance of the site, but there's a trailing comma here.